UltiSnips/elixir.snippets

```diff
@@ -0,0 +1,126 @@
+snippet if
+if ${1:condition} do
+	${2:expression}
+${3:end}
```

Comment:
I think remembering that `end` is a group is more to remember than that you can type `else` at any time. I could imagine tying a larger block of code inside the `if` and then stumbling over the fact the `end` has been selected when I press tab later.

Pretty much just what's already been made. I've been trying to clean up some of the languages, but honestly there's still rather a lot of mess.